server/src/main/java/com/paypal/selion/grid/servlets/SauceConfigChangeServlet.java

```diff
@@ -148,4 +170,15 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
 
         ServletHelper.respondAsHtmlWithMessage(resp, msg);
     }
+
+    private void addToJsonObject(JsonObject jsonObject, String key, Object value) {
+        if (value instanceof String) {
+            if (StringUtils.isNotBlank((String) value)) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2439971405/issues/source?bid=2928425&fileBranchId=3233060#l176)